Sources/protoc-gen-swift/MessageFieldGenerator.swift

```diff
@@ -275,10 +275,12 @@ extension Google_Protobuf_FieldDescriptorProto {
 struct MessageFieldGenerator {
     private let fieldDescriptor: FieldDescriptor
     private let generatorOptions: GeneratorOptions
+    private let namer: SwiftProtobufNamer
 
     var descriptor: Google_Protobuf_FieldDescriptorProto { return fieldDescriptor.proto }
     let oneof: Google_Protobuf_OneofDescriptorProto?
     let jsonName: String?
+    let hasFieldPresence: Bool
```

Comment:
Proto generation for a lot of Google's products is a non trivial part of a lot of our builds.  Anyway, this is just a checkpoint to get the namer support in for fields.  I'm still working on a pretty large change to how the message/fields generate so a lot of this will change in follow ups.